l2tscaffolder/scaffolders/turbinia.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""Turbinia component scaffolder."""
+import datetime
+import os
+import logging
+
+from typing import Dict
+from typing import Iterator
+from typing import Tuple
+
+from l2tscaffolder.lib import definitions
+from l2tscaffolder.lib import mapping_helper
+from l2tscaffolder.scaffolders import interface
+
+
+class TurbiniaBaseScaffolder(interface.Scaffolder):
+  """The Turbinia base scaffolder interface.
+
+  Attributes:
+    class_name (str): class name of the Turbinia plugin to be generated.
+  """
+
+  # The name of the plugin this scaffolder plugin provides.
+  NAME = 'turbinia_base'
+
+  # One liner describing what the scaffolder provides.
+  DESCRIPTION = 'This is a scaffolder for Turbinia components'
+
+  # Define which project this particular scaffolder belongs to.
+  PROJECT = definitions.DEFINITION_TURBINIA
+
+  # Filename of templates.
+  TEMPLATE_JOB_FILE = None
+  TEMPLATE_TASK_FILE = None
+
+  def __init__(self):
+    """Initializes the Turbinia scaffolder."""
+    super(TurbiniaBaseScaffolder, self).__init__()
+    self._plugin_path = os.path.join('turbinia', 'jobs')
```

Comment:
Please change all the references from "plugin" to "job" to make it clear what's actually being generated here.